knative-operator/deploy/resources/dashboards/grafana-dash-knative-eventing-source.yaml

```diff
@@ -589,7 +589,7 @@ data:
           "thresholds": "",
           "timeFrom": null,
           "timeShift": null,
-          "title": "MT PingSource: Success Rate  (2xx Event, fraction rate per minute)",
+          "title": "MT PingSource: Success Rate  (2xx Event, fraction rate, over 1m window)",
```

Comment:
@skonto The last instance of double spacing between "Success Rate" and "(2xx...".